apps/api/src/app/integrations/integrations.controller.ts

```diff
@@ -70,6 +74,26 @@ export class IntegrationsController {
     );
   }
 
+  @Get('/get-webhook-support-status/:channel/:providerId')
```

Comment:
I do have some personal considerations regarding the proposed endpoint path.
One is that I feel like having to pass the channel and the provider is like duplicating the information because knowing the provider we know which channel is being used (either from database or by domain in the code). Also passing 2 would force us to double check that wrong combinations of channel and provider are not made.
Then adding the HTTP method in the naming of the endpoint doesn't feel right based in REST design.

My suggestion would be:
```suggestion
  @Get('/webhooks/provider/:providerId/status')
```
The reasons are:
- feels like webhooks can be a URL path very reusable. And the addition of support with my knowledge of our API and what's trying to be done here doesn't make me feel gives extra information.
- as the webhook entity in our system is the parent, the child is always going to be a provider, so the first subresource should be based in REST conventions the subresource name `providers` and the id of the provider.
- finally we request the specified property of the subresource, the status of the provider

What do you think about it?

Reply:
Agreed, will update it, Thanks